app/controllers/access_tokens_controller.rb

```diff
@@ -7,8 +7,8 @@ def create
         grant_type: 'authorization_code',
         code: params[:code],
         redirect_uri: 'http://localhost:3000/access_tokens',
-        client_id: 'a2f62fa1d6764016a0e2ef1f8d6f0428',
-        client_secret: 'client_secret'})
+        client_id: Rails.application.config.spotify_client_id,
```

Comment:
By making these application configs, you wont have to remember to remove it every time you push to github